templates/config/70_nodepools.yaml.j2

```diff
@@ -0,0 +1,38 @@
+---
+nodepools:
+  abstract:
+    mech-ansible:
+      mechanism:
+        type: "ansible"
+        ansible:
+          topdir: "/home/duffy/src/duffy-ansible"
+          extra-vars:
+{% raw %}
+            nodepool: "{{ name }}"
+            template_name: "{{ name }}"
+{% endraw %}
+    virtual:
+      extends: "mech-ansible"
+      reuse-nodes: false
+      mechanism:
+        ansible:
+          extra-vars:
+            api_url: "http://n35.hufty.ci.centos.org:2633"
```

Comment:
unsure about that but clearly not a static string that needs to be defined, and should be a variable ?

Reply:
Indeed! :see_no_evil: